src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -84,9 +84,12 @@ static void LaunchAndWait(Permutation permutation)
                 {
                     p.Kill();
                     Assert.Fail("Killed!");
```

Comment:
Can't this be a little more descriptive? As in, I waited x-seconds and the process still seems to be working. Make it complete faster, please...